lib/ohai/config.rb

```diff
@@ -16,28 +18,91 @@
 # limitations under the License.
 #
 
-require 'mixlib/config'
+require 'ohai/log'
+require 'chef-config/logger'
+
+ChefConfig::Logger = Ohai::Log
+
+require 'chef-config/config'
 
 module Ohai
+  Config = ChefConfig::Config
```

Comment:
Adds these settings to ChefConfig::Config. See: https://github.com/chef/chef/blob/master/lib/chef/config.rb

I will add a comment.